beets/util/__init__.py

```diff
@@ -222,6 +222,10 @@ def sorted_walk(path, ignore=(), ignore_hidden=False, logger=None):
         for res in sorted_walk(cur, ignore, ignore_hidden, logger):
             yield res
 
+def pathlib_as_posix(path):
```

Comment:
I think this might be more sensibly called `path_as_posix`? (I think it's simple enough that we don't have to give credit to pathlib. :smiley:)